src/games/strategy/triplea/ui/AbstractForumPosterPanel.java

```diff
@@ -21,13 +25,23 @@
   protected IPlayerBridge m_bridge;
   protected PBEMMessagePoster m_poster;
   protected TripleAFrame m_frame;
-  protected Action m_doneAction;
   protected ForumPosterComponent m_forumPosterComponent;
+  protected AbstractAction m_doneAction = new AbstractAction("Done") {
+    private static final long serialVersionUID = -5084680807785728744L;
+
+    @Override
+    public void actionPerformed(final ActionEvent e) {
+      if (m_forumPosterComponent.getHasPostedTurnSummary() || JOptionPane.YES_OPTION ==
```

Comment:
The getHasPosted... is to determine whether or not we have posted. That is the entire purpose of this PR - to prevent inadvertant clicks of the "Done" button without posting. I still think that is self explanatory.